test/unit/migration/KnexMigrationSource.test.ts

```diff
@@ -54,7 +54,7 @@ describe('MIGRATION: KnexMigrationSource', () => {
       const instance = await getInstance();
       const dbStub = {} as any;
 
-      const migration = instance.getMigration('mgr_002');
+      const migration = await instance.getMigration('mgr_002');
```

Comment:
Hmm ... there is something off here. Might not need `getMigration` to return a `Promise`.

Reply:
`getMigration()` type definition says it returns a promise 😅

Reply:
Understood. The type definition is a little off than the documentation. Cool!